lib/muffin_man/notifications/v1.rb

```diff
@@ -0,0 +1,63 @@
+module MuffinMan
+  module Notifications
+    require "json"
+    require "sp_api_helpers"
+    class V1 < SpApiClient
+
+      NOTIFICATION_PATH = "/notifications/v1"
+
+      def create_destinations(arn, params={})
+        params = params.transform_keys{|key| key.to_s}
+        @local_var_path = "#{NOTIFICATION_PATH}/destinations"
+        destination_params = {"resourceSpecification": {"sqs": {"arn": arn}}, "marketplaceIds": params["marketplace_ids"]}
+        destination_params.merge!("eventBridge": {"region": params["region"], "accountId": params["account_id"]}) unless params["region"].nil? || params["account_id"].nil?
+        destination_params.merge!("name": params["name"]) unless params["name"].nil?
+        @request_body = destination_params
+        @request_type = "POST"
+        call_api
+      end
+
+       def get_destinations(params={})
+        @local_var_path = "#{NOTIFICATION_PATH}/destinations"
+        @query_params = sp_api_params(params)
+        @request_type = "GET"
+        call_api
+      end
+
+      def get_destination(destination_id)
+        @local_var_path = "#{NOTIFICATION_PATH}/destinations/#{destination_id}"
+        @request_type = "GET"
+        call_api
+      end
+
+      def create_subscriptions(notification_type, params={})
+        params = params.transform_keys{|key| key.to_s}
+        @local_var_path = "#{NOTIFICATION_PATH}/subscriptions/#{notification_type}"
+        subscription_params = {"destinationId": params["destination_id"], "processingDirective": {"eventFilter": {"eventFilterType": notification_type, "marketplaceIds": params["marketplace_ids"]}}}
```

Comment:
Looks like there is an `aggregationSettings` param in the `processingDirective` that we're not handling here

Reply:
It is optional and not required for existing flow so did not add this earlier. Added now.